docs/infrastructure/logging.md

```diff
@@ -387,3 +398,62 @@ For example, in the above example `project-manager` will log events of up-to
 
 **Note** Logging to a file requires presence of the `file`
 [appender](#file-appender) in the `logging-service.appenders` section.
+
+# How to use logging
+
+Logging infrastructure uses a popular SLF4J interface which most of developers
+should be familiar with. In this section we include a only small number of
+examples, full user manual is available at SLF4J's
+[website](https://www.slf4j.org/manual.html).
+
+## Log a simple INFO message
+
+```
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class HelloWorld {
+
+  public static void main(String[] args) {
+    Logger logger = LoggerFactory.getLogger(HelloWorld.class);
+    logger.info("Hello World");
+  }
+}
+```
+
+## Log a simple INFO message only if TRACE is enabled
+
+```
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class HelloWorld {
+
+  public static void main(String[] args) {
+    Logger logger = LoggerFactory.getLogger(HelloWorld.class);
+    if (logger.isTraceEnabled()) {
+      logger.info("Hello World");
+    }
+  }
+}
+```
+
+## Log an exception
+
+```
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class HelloWorld {
+
+  public static void main(String[] args) {
```

Comment:
Again, example of logging from Enso (including how to launch the CLI) would be more useful than standalone class unrelated to Enso. Is it guaranteed that `./enso --run hello.enso` would print exception logged this way to console? Is there a test for that?

Reply:
You clearly haven't read the documentation. We **don't** log stacktraces in console, on purpose. You can of course change that by removing `%noexp` but we are not going to change that for users. It's all documented.

Reply:
> We don't log stacktraces in console, on purpose.

Then your answer should be: _It is not possible._ I am not OK with such an answer, but that's a unrelated to the answer.